jenkins_pipelines/manager_prs/Jenkinsfile_testsuite

```diff
@@ -6,11 +6,12 @@ properties([
     disableConcurrentBuilds(),
 ])
 
+def deployed = false
+
 pipeline {
 
     parameters {
         string(defaultValue: 'Manager-3.2', description: 'Testsuite GitHub branch: It should match the value at manager-3.2-2obs', name: 'testsuite_branch')
-        string(defaultValue: '32/VARS-full-NUE.sh', description: 'Sumaform Test Runner environment to use', name: 'sumaform_env')
```

Comment:
Why?

Reply:
> LGTM. Just one question... and something I just noticed: we have this at `manager_prs ` folder, but it doesn't make much sense (anyway fixing it is not for this PR).

At that moment it only runs on Manager-3.2, it supposed to be identical for Uyuni, parametrizing it properly. But let's keep it here, until we do the transition to Uyuni.